lib/Service/Mail/Mailer.hs

```diff
@@ -111,15 +111,15 @@ loadMailTemplateParts mailName context = do
     then do
       templateFileParts <- loadFileParts $ root ++ _MAIL_TEMPLATE_ATTACHMENTS_FOLDER
       globalFileParts <- loadFileParts $ commonRoot ++ _MAIL_TEMPLATE_ATTACHMENTS_FOLDER
-      return $ mainParts ++ templateFileParts ++ globalFileParts
-    else return []
+      return . Right $ mainParts ++ fromRight [] templateFileParts ++ fromRight [] globalFileParts
+    else return $ Left "No HTML neither PLAIN TEXT"
```

Comment:
Please put it to the Localization file

Reply:
Fixed in https://github.com/ds-wizard/dsw-server/pull/56/commits/9138cc713207b11a3590d895176d5d1558b8b19d